src/v2/Components/ArtworkFilter/SavedSearch/Components/CreateAlertButton.tsx

```diff
@@ -10,9 +12,20 @@ interface CreateAlertButtonProps {
 export const CreateAlertButton: React.FC<CreateAlertButtonProps> = ({
   savedSearchAttributes,
 }) => {
+  const tracking = useTracking()
   const [visibleForm, setVisibleForm] = useState(false)
 
-  const handleOpenForm = () => setVisibleForm(true)
+  const trackInfo = {
+    action: ActionType.clickedCreateAlert,
+    context_page_owner_type: OwnerType.artist,
+    context_page_owner_id: savedSearchAttributes.id,
+    context_page_owner_slug: savedSearchAttributes.slug,
+  }
+
+  const handleOpenForm = () => {
+    setVisibleForm(true)
+    tracking.trackEvent(trackInfo as any)
```

Comment:
**suggestion**: maybe it would be better to use this option instead of any?

```javascript
tracking.trackEvent({
  action: ActionType.clickedCreateAlert,
  context_page_owner_type: OwnerType.artist,
  context_page_owner_id: savedSearchAttributes.id,
  context_page_owner_slug: savedSearchAttributes.slug,
})
```